docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -74,24 +100,64 @@ Below is a list of separate examples for different frameworks and Model Optimize
 
 1. Launch Model Optimizer for a TensorFlow MobileNet model in the binary protobuf format:
 
-   .. code-block:: sh
-
-      mo --input_model MobileNet.pb
+   .. tab-set::
+   
+       .. tab-item:: MO command line tool
+          :sync: cli-tool
+   
+          .. code-block:: sh
+   
+             mo --input_model MobileNet.pb
+   
+       .. tab-item:: MO Python API
+          :sync: mo-python-api
+   
+          .. code-block:: python
+   
+             from openvino.tools.mo import convert_model
+             ov_model = convert_model("MobileNet.pb")
 
 
    Launch Model Optimizer for a TensorFlow BERT model in the SavedModel format with three inputs. Specify input shapes explicitly where the batch size and the sequence length equal 2 and 30 respectively:
 
-   .. code-block:: sh
-
-      mo --saved_model_dir BERT --input mask,word_ids,type_ids --input_shape [2,30],[2,30],[2,30]
-
-      For more information, refer to the :doc:`Converting a TensorFlow Model <openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_TensorFlow>` guide.
+   .. tab-set::
+   
+       .. tab-item:: MO command line tool
+          :sync: cli-tool
+   
+          .. code-block:: sh
+   
+             mo --saved_model_dir BERT --input mask,word_ids,type_ids --input_shape [2,30],[2,30],[2,30]
+   
+       .. tab-item:: MO Python API
+          :sync: mo-python-api
+   
+          .. code-block:: python
+   
+             from openvino.tools.mo import convert_model
+             ov_model = convert_model(saved_model_dir="BERT", input=["mask", "word_ids", "type_ids"], input_shape=[[2,30],[2,30],[2,30]])
```

Comment:
@sgolebiewski-intel did you check these commands? 
Bert model is expected to not work with batch **2**. So most likely shapes here need to be changed.

Reply:
Yes, confronted it with what you have proposed in :

https://github.com/openvinotoolkit/openvino/pull/16761/files#diff-6a3db6969c80ae509adac48b7a6b3523fb542f7f74ee33d2fc95fa8f999ef909

:)

Reply:
Yes, I wrote the identical python API command to original CLI command. But I didn't check that original command still works. We observed that some commands stopped to work, so we need to check them. 

Can you please take a bert model and check that command from this page work?
If you will have troubles with finding the model contact me or @rkazants